File: iHappy/iHappy/Bizhi/Model/YSEImageModel.m
//
//  YSEImageModel.m
//  iHappy
//
//  Created by dusheng.xu on 2017/5/14.
//  Copyright © 2017年 上海优蜜科技有限公司. All rights reserved.
//

#import "YSEImageModel.h"

@implementation YSEImageModel

- (instancetype)init
{
    self = [super init];
    if (self) {
        self.title = @"";
        self.href = @"";
        self.width = @"";
        self.height = @"";
    }
    return self;
}

- (void)p_setTitle:(NSString *)title href:(NSString *)href width:(NSString *)width height:(NSString *)height{
    self.title = title;
    self.href = href;
    self.width = width;
    self.height = height;
}

@end

File: youse壁纸站/youse/ViewControllers/main/Model/YSEImageModel.swift
//
//  YSEImageModel.swift
//  youse
//
//  Created by xudosom on 16/9/11.
//  Copyright © 2016年 上海优蜜科技有限公司. All rights reserved.
//

import UIKit

class YSEImageModel: NSObject {
    fileprivate(set) var title:String?;
    fileprivate(set) var href:String?;
    fileprivate(set) var width:String?;
    fileprivate(set) var height:String?;
    
    override init() {
        super.init();
        self.title = "";
        self.href = "";
        self.width = String(CGFloat.leastNormalMagnitude);
        self.height = String(CGFloat.leastNormalMagnitude);
    }
    
    internal func p_setTitle(_ title:String?, href:String?, width:String?, height:String?){
        self.title = title;
        self.href = href;
        self.width = width;
        self.height = height;
    }
}
